src/include/txn.i

```diff
@@ -193,6 +193,166 @@ __wt_timestamp_subone(wt_timestamp_t *ts)
 
 #endif /* HAVE_TIMESTAMPS */
 
+/*
+ * __txn_op_modify_recno --
```

Comment:
Could you call this `set_recno` rather than `modify_recno` - we have WT_MODIFY structures, so the modify word can be confusing.